### ISSUE TITLE
YZ Joiner cable hole problem with crossbeam style walls

### ISSUE THREAD

**Issue by ArktekniK** [revarbat/snappy-reprap#3]
Hi,
I am experiencing lots of shrinkage with the default STLs, so I switch the wall styling and noticed YZ joiner has a problem with crossbeam style walls. The hole is too large and causes the beams not to meet. To be honest, I don't think the hole is needed if you've chosen crossbeam walls.

Lines 98-100 of YZ joiner parts should be wrapped in a conditional
https://github.com/revarbat/snappy-reprap/blob/master/yz_joiner_parts.scad#L98#L100

Something like:

```
// Wiring access hole
if (wall_style != "crossbeams") {
    up(rail_thick+rail_height/4/2) {
      cube([16, joiner_width*3, rail_height/4], center=true);
    }
}
```

**Comment by ArktekniK** [comment]
![yz hole](https://cloud.githubusercontent.com/assets/9900921/10117864/1d802994-645e-11e5-9aa1-e03ffde85d26.png)

**Comment by revarbat** [comment]
Good catch.

**Comment by revarbat** [comment]
[v1.1 0082da0] Fixed issue #3: YZ Joiner cable hole problem with crossbeam style walls
 1 file changed, 4 insertions(+), 2 deletions(-)
